dashbuilder-client/dashbuilder-dataset-editor/src/main/java/org/dashbuilder/dataset/editor/client/screens/DataSetDefWizardScreen.java

```diff
@@ -216,10 +216,10 @@ void onSave(final DataSetDef dataSetDef, final String message) {
 
     RemoteCallback<Path> saveSuccessCallback = new RemoteCallback<Path>() {
         @Override public void callback(Path path) {
+            currentWorkflow.clear();
```

Comment:
The NPE is present in both. The only difference in that kie-wb is propagating the uncatched exception to the UI and dashbuilder is not. No idea why, I haven't investigated  much to be honest. The NPE is fixed anyway.